src/xml/ConfigParser.cpp

```diff
@@ -59,14 +59,14 @@ void OnStartElementNs(
     auto        valueEnd   = reinterpret_cast<const char *>(attributes[index + 4]);
     std::string value(valueBegin, valueEnd);
 
-    attributesMap[attributeName] = decodeXML(value);
+    attributesMap[attributeName] = decodeXML(std::move(value));
```

Comment:
This is a strong indicator that `decodeXML` should take a `string_view` and return a decoded owning `std::string`.